docs/example-react-hooks-useReducer.md

```diff
@@ -0,0 +1,78 @@
+Basic example showing how to test useReducer hook. 
+
+
+The component we are testing which changes some text depending on `stateprop1` value.
+
+```jsx
+import React, { useReducer } from 'react'
+
+const initialState = {
+    isConfirmed: false,
+}
+
+function reducer(state = initialState, action) {
+  switch(action.type) {
+    case "SUCCESS":
+      return {
+        ...state,
+        isConfirmed: true,
+      }
+    case "FAILURE":
+      return {
+        ...state,
+        isConfirmed: true,
+      }
+    default:
+      console.log(state)
+  }
+}
+
+const Example = () => {
+  const [state, dispatch] = useReducer(reducer, initialState)
+
+  const dispatchActionSuccess = () => {
+    dispatch({ type: 'SUCCESS' })
+  }
+
+  const dispatchActionFailure = () => {
+    dispatch({ type: 'FAILURE' })
+  }
+
+  return (
+    <div>
+       <div>
```

Comment:
the wrapper `<div>` feels unnecessary:

```jsx
<p>
  {state.isConfirmed
    ? 'Confirmed!'
    : 'Waiting for confirmation...'}
</p> 
```

(sry I can't provide multiline suggestions 😞)